ykllvmwrap/build.rs

```diff
@@ -38,8 +80,19 @@ fn main() {
     if env::var("PROFILE").unwrap() == "release" {
         comp.flag("-DNDEBUG");
     }
+
+    // We use a compiler wrapper to capture compilation commands and generate a
+    // `compile_commands.json` database for clangd.
+    let td = TempDir::new().unwrap();
+    env::set_var("YKLLVMWRAP_JSON_TEMP", td.path());
```

Comment:
I guess there's no way to do this *just* for the ykllvmwrap subprocess?

Reply:
Afraid not. I had hoped for a `compiler.env(key, val)`, but there isn't one.